Day4Homework/src/main/kotlin/solutionB.kt

```diff
@@ -0,0 +1,16 @@
+fun main() {
+
+
+    var v = FindVowel("jumanah")
+    println(v)
```

Comment:
You don’t need to use println here because FindVowel() is unit or void. Therefore, it will print the type of the function not the value